components/routine-logs-view.js

```diff
@@ -18,13 +21,17 @@ const columns = [
   { field: "message", headerName: "Message", flex: 1, },
 ];
 
-export default function RoutineLogsView( {logsPerPage} ) {
+export default function RoutineLogsView({ logsPerPage }) {
   const selectedRoutine = useRecoilValue(selectedRoutineState);
   const [open, setOpen] = useState(false);
   const [time, setTime] = useState("");
   const [level, setLevel] = useState("");
   const [message, setMessage] = useState("");
-  const [logs, setLogs] = useState([]);
+
+  const { data } = useSWR(selectedRoutine ? `routine_logs/${selectedRoutine}` : null, apiFetch, {
+    refreshInterval: 5000,
```

Comment:
Move to const

Reply:
I'd suggest allowing the user to configure it via UI. 
What do you think?

Reply:
IMO I don't think we should implement it if we don't need it yet :)

Reply:
Yes, it might be better to implement it as a separate feature.